spec/features/collection_drafts/forms/spatial_information_form_spec.rb

```diff
@@ -122,6 +122,12 @@
         expect(page).to have_content('Collection Draft Updated Successfully!')
       end
 
+      it 'contains the correct dropdown options' do
+        within '#draft_tiling_identification_systems_0_tiling_identification_system_name' do
+          page.has_select?('Tiling Identification System Name', options: ['CALIPSO', 'MISR', 'MODIS Tile EASE', 'MODIS Tile SIN', 'WELD Alaska Tile', 'WELD CONUS Tile', 'WRS-1', 'WRS-2', 'Military Grid Reference System'])
```

Comment:
this should follow how all the other tests are written - `expect(page).to have_select(...`

Reply:
@charleshuang80 That was my first thought when you suggested using this matcher, but as I tried to adapt it to the `expect(page).to` style, my tests kept failing. I've tried `expect(page).to have_select?('Tiling Identification System Name', options: ...` and `expect(page).to has_select?('Tiling Identification System Name', options: ...` with no success. I think that matcher only works as it is above